condor_watch_q.py

```diff
@@ -216,6 +217,26 @@ def parse_args():
     return args
 
 
+def parse_unknown_args(unknown):
+    for unknown_arg in unknown:
+        print("Unknown command.", end=" ")
+        if unknown_arg.isdigit():
+            print("Did you mean condor_watch_q -clusters {}?".format(unknown_arg))
```

Comment:
Add a space before the `?` in each of these so that it's easier to copy-paste it if needed.